pom.xml

```diff
@@ -89,6 +89,26 @@
             <version>${vertx.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
should these be removed?

Reply:
Right, we'll remove these dependencies for now while I look into reworking loki